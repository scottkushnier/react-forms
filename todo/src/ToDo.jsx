
function ToDo({text, deleteFn}) {
    return (
        <div>
           <button onClick={deleteFn}>x</button>
           <div className='todo'>
           &nbsp; {text}
          </div>
        </div>
    )
  }

  export default ToDo;