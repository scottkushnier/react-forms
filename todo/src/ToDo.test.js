
import { render } from "@testing-library/react";
import ToDo from "./ToDo";

test("colorbox smoke test", function () {
  render(<ToDo text="laundry"/>)
});

test("matches snapshot", function() {
    const {asFragment} = render(<ToDo text='shopping'/>);
    expect(asFragment()).toMatchSnapshot();
  });

