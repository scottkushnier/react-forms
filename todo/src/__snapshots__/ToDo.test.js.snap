// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <button>
      x
    </button>
    <div
      class="todo"
    >
      Â  shopping
    </div>
  </div>
</DocumentFragment>
`;
